portal-impl/src/com/liferay/portal/verify/VerifyProperties.java

```diff
@@ -1580,7 +1581,8 @@ protected void verifySystemProperties() throws Exception {
 		"browser.compatibility.ie.versions",
 		"buffered.increment.parallel.queue.size",
 		"buffered.increment.serial.queue.size",
-		"cache.clear.on.context.initialization", "captcha.max.challenges",
+		"cache.clear.on.context.initialization",
+		"Calendar.publish.to.live.by.default", "captcha.max.challenges",
```

Comment:
@jeffwu0724 The first character should be lower cased.

Reply:
Hey Tina, sorry for the mistakes. Already fixed